docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.md

```diff
@@ -63,10 +61,9 @@ stores:
           base_directory: shared_expectations/
 ```
 
+## 4. Confirm that the new Expectation Suites have been added
 
-### 4. Confirm that the new Expectation Suites have been added
-
-To confirm that Great Expectations can find your Expectation Suites, run the following Python code:
+If you copied your existing Expectation Suites to your filesystem, run the following Python command to confirm that Great Expectations can find them:
 
 ```python
```

Comment:
3rd instance of this code block.  They can all share the same example script, so we should flag them as needing to be under test.  Frankly: I'd be surprised if it doesn't already exist in one example script or another, it just needs updated there to be wrapped in a named snippet and then updated here to use that snippet.

Reply:
@Rachel-Reverie this has been noted.